cmake/mkl/MKLConfig.cmake

```
@@ -208,12 +208,28 @@ if(CMAKE_Fortran_COMPILER)
 endif()
 
 # Determine Compiler Family
-if(CXX_COMPILER_NAME STREQUAL "dpcpp" OR CXX_COMPILER_NAME STREQUAL "dpcpp.exe"
-    OR CXX_COMPILER_NAME STREQUAL "icpx" OR CXX_COMPILER_NAME STREQUAL "icx.exe")
-  set(SYCL_COMPILER ON)
+
+include(CMakePackageConfigHelpers)
+include(CheckCXXCompilerFlag)
+include(CheckIncludeFileCXX)
+include(GNUInstallDirs)
```

```suggestion
include(CheckCXXCompilerFlag)
include(CheckIncludeFileCXX)
```
I think we can delete the first line `include(CMakePackageConfigHelpers)` and the fourth line `include(GNUInstallDirs)`. `CheckCXXCompilerFlag` is needed for `check_cxx_compiler_flag()`, and `CheckIncludeFileCXX` is needed for `CHECK_INCLUDE_FILE_CXX()`.